shell/platform/fuchsia/dart_runner/meta/dart_aot_product_runner.cmx

```diff
@@ -4,6 +4,7 @@
   },
   "sandbox": {
     "features": [
+	  "config-data",
```

Comment:
can we indent this to be in accordance with the other lines in this file? Same for other cmx files too.

Reply:
Yes. That reminded me to configure my editor correctly, too.